app/styles/app.scss

```diff
@@ -18,6 +18,7 @@
 @import "goods_details";
 @import "appointment_details";
 
+@import "node_modules/ember-searchable-select/app/styles/ember-searchable-select/style";
```

Comment:
I think sass node_modules should be imported from ember-cli-build.js ... but I'm not sure
@Darshan-Chauhan would you know ?

Reply:
@patrixr , I agree. I would prefer third party imports in `ember-cli-build`.

Reply:
@patrixr @Darshan-Chauhan i moved above import to ember-cli-build and added extension .scss and restarted the server after that css is not getting applied to dropdown fields

Reply:
So keeping it in the same file only.